scripts/flash/approve-account.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var approveAugurEternalApprovalValue = require("../dp/lib/approve-augur-eternal-approval-value");
+
+function approveInternal(augur, address, universe, auth, callback) {
+  console.log(chalk.green.dim("address:"), chalk.green(address));
+  console.log(chalk.green.dim("universe:"), chalk.green(universe));
+  approveAugurEternalApprovalValue(augur, address, auth, function (err) {
+    if (err) {
+      console.log(chalk.red(err));
+      return callback(JSON.stringify(err));
+    }
+    callback(null);
+  });
+}
+
+function help(callback) {
```

Comment:
Why does `help` require a `callback` argument?  It does not perform any asynchronous operations.

Reply:
the call to the script is passed to help() a callback needs to be called to tell index.js the script is done running. index.js doesn't know help is called it's just executing the script.